Cargo.toml

```diff
@@ -4,4 +4,8 @@ members = [
 	"webrender_traits",
 	"sample",
 	"replay",
+	"wrench",
 ]
+
+[profile.release]
+debug = true
```

Comment:
Does this have any effect on runtime performance? Should we just enable it when profiling?